src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.Interactive_api_is_not_changed.approved.txt

```diff
@@ -568,10 +568,6 @@ Microsoft.DotNet.Interactive.Events
     .ctor(Microsoft.DotNet.Interactive.Commands.SubmitCode command)
     public System.String Code { get;}
     public System.String ToString()
-  public class CommandCancelled : KernelEvent
```

Comment:
I'm hoping nothing external was depending on this :)

Reply:
But glad to see it go away - should make it easier to test for failure states. #2985